app/javascript/dashboard/routes/dashboard/conversation/labels/LabelBox.vue

```diff
@@ -100,19 +124,32 @@ export default {
         // Ignore error
       }
     },
-    onEdit() {
-      this.isEditing = true;
+
+    toggleLabels() {
+      this.showSearchDropdownLabel = !this.showSearchDropdownLabel;
     },
-    closeEditModal() {
-      bus.$emit('fetch_conversation_stats');
-      this.isEditing = false;
+
+    closeDropdownLabel() {
+      this.showSearchDropdownLabel = false;
```

Comment:
Can we rename this to `showSearchDropdownLabel` to `showLabelSelector` or something like that ?